### DISCUSSION TITLE
Deleting multiple S3 objects by wildcard

### DISCUSSION THREAD

**Discussion by indreksiitan** [async-aws/aws#1669] [Q&A] [answered]
Can anyone enlighten me on what is the correct call syntax to delete multiple objects at once in the S3 client? I've tried all I could imagine from common sense and reading the source, but nothing works. I either get an error message (that "Objects" or "Delete" parameters are required - and cannot understand from the code what they should contain exactly) or no error, but nothing deleted. Deleting a single object using Bucket and Key parameters works fine.

**Comment by stof** [comment, marked as answer]
The AWS API endpoint for deleting objects does not support wildcards. That's why it is not supported in our SDK.
